src/character.cpp

```diff
@@ -5651,7 +5635,7 @@ std::pair<int, int> Character::weariness_transition_progress() const
 
 int Character::weariness_level() const
 {
-    int amount = weariness();
+    int amount = activity_history.weariness();
```

Comment:
Seems like with the change to `Character::weariness()` this could be left alone.